pokemongo_bot/cell_workers/find_biggest_cluster.py

```diff
@@ -0,0 +1,40 @@
+from pokemongo_bot.cell_workers.utils import distance, merc2coord, coord2merc
+from networkx.algorithms.clique import find_cliques
+
+import networkx as nx
+import numpy as np
+
+
+class FindBiggestCluster(object):
```

Comment:
Are you able to combine the FindBiggestCluster and FollowBiggestCluster? Our other navigators are only a single task. Perhaps you can take a look at those for examples.